### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by fidelman** [event-maker-telegram-bot/event-maker-telegram-bot#17]
Logo for Github Organisation, Telegram Avatar and so on. I think, it will be very nice to have branded bot.
I've prepared a few example, what do you think about a concept, colors can be done later
<img width="383" alt="Screenshot 2020-10-05 at 14 32 50" src="https://user-images.githubusercontent.com/15198653/95080065-d094ff80-0717-11eb-810b-ee6187cad88f.png">
<img width="388" alt="Screenshot 2020-10-05 at 14 32 22" src="https://user-images.githubusercontent.com/15198653/95080070-d1c62c80-0717-11eb-9766-305a0f331bc9.png">

**Comment by bmind12** [comment]
@fidelman I like the first one, the second is too pretentious for this :)

**Comment by fidelman** [comment]
> @fidelman I like the first one, the second is too pathetic for this :)

agree, I will ask Karina to adapt colors that it looks like really Telegram bot
